webapp/src/main/webapp/news-list-view/components/views/NewsMosaicView.vue

```diff
@@ -147,6 +147,13 @@ export default {
         seeAllUrl: this.seeAllUrl,
       };
     },
+    illustrationURLResize(itemIllustrationURL) {
+      const element = document.querySelector('#top-news-mosaic .articleLink');
```

Comment:
How it work if you have more than one .articlelink in #top-news-mosaic ?
I'm afraid that, for example, the second, third ... news take the same size as the first one.
Should we use the idItem ?

Reply:
It’s true, I tested with only one news.